internal/controller/trigger/controller.go

```diff
@@ -503,3 +507,32 @@ func (ctrl *controller) Stop(ctx context.Context) {
 		})
 	}
 }
+
+func (ctrl *controller) initTriggerSystemEventbus() {
+	// avoid blocking starting
+	go func() {
+		ctx := context.Background()
+		log.Info(ctx, "trigger controller is ready to check system eventbus", nil)
+		if err := ctrl.cl.WaitForControllerReady(true); err != nil {
+			log.Error(context.Background(), "trigger controller try to create system eventbus, "+
+				"but Vanus cluster hasn't ready, exit", nil)
+			os.Exit(-1)
+		}
+
+		if err := ctrl.cl.EventbusService().CreateSystemEventbusIfNotExist(ctx, primitive.RetryEventbusName,
+			"System Eventbus For Trigger Service"); err != nil {
+			log.Error(context.Background(), "failed to create RetryEventbus, exit", map[string]interface{}{
+				log.KeyError: err,
+			})
+			os.Exit(-1)
```

Comment:
Why do need to exit when the creation of the retry eventbus fails, but the deadletter message eventbus does not